project/app/src/main/java/org/owntracks/android/model/messages/MessageLocation.kt

```diff
@@ -10,6 +10,9 @@ import java.lang.ref.WeakReference
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 open class MessageLocation(private val dep: MessageWithCreatedAt = MessageCreatedAtNow(RealClock())) : MessageBase(), MessageWithCreatedAt by dep {
+    @JsonIgnore
+    override val numberOfRetries: Int = 10000 // This should last a week at 1 attempt per minute
```

Comment:
Do you want to make this a full week? At once/minute, 10080 is a week.

Reply:
I rounded down :p

I guess we should be true to a human spec, not a base-10 arbitrary spec.